be/src/olap/schema_change.cpp

```diff
@@ -1772,6 +1773,7 @@ OLAPStatus SchemaChangeHandler::schema_version_convert(
         SegmentGroup* new_segment_group = nullptr;
         if ((*it)->transaction_id() == 0) {
             new_segment_group = new SegmentGroup(dest_tablet->tablet_id(),
+                                                 0,
```

Comment:
why not generate a random rowsetid here?

Reply:
It is only to make it can be compiled. This function will changed in the following version.
I must make it all can be compiled before changing it.